client/sites-dashboard-v2/sites-dataviews/index.tsx

```diff
@@ -61,6 +61,29 @@ const DotcomSitesDataViews = ( {
 		[ setDataViewsState ]
 	);
 
+	useEffect( () => {
```

Comment:
Cool workaround! I asked ChatGPT to improve it and it recommended adding the event listener to a closer parent than document, for better performance:
 
```
const rowsContainer = document.querySelector('.dataviews-view-table');
    if (rowsContainer) {
        rowsContainer.addEventListener('click', handleRowClick);
    }

    return () => {
        if (rowsContainer) {
            rowsContainer.removeEventListener('click', handleRowClick);
        }
    };
```

Reply:
Sounds good!